src/plugins/flexibleLayout/components/flexibleLayout.vue

```diff
@@ -10,40 +10,31 @@
                 'c-fl--rows': rowsLayout === true
             }">
 
-            <div class="u-contents"
-                 v-for="(container, index) in containers"
-                 :key="index">
+            <template v-for="(container, index) in containers">
                 
                 <drop-hint
                     style="flex-basis: 15px;"
                     v-if="index === 0 && containers.length > 1"
-                    v-show="isContainerDragging"
+                    :key="index"
                     :index="-1"
+                    :allow-drop="allowContainerDrop"
                     @object-drop-to="containerDropTo">
                 </drop-hint>
 
                 <container-component
                     class="c-fl__container"
-                    ref="containerComponent"
+                    :key="container.id"
                     :index="index"
-                    :size="`${Math.round(container.width)}%`"
-                    :frames="container.frames"
-                    :isEditing="isEditing"
-                    :isDragging="isDragging"
+                    :sizeString="`${Math.round(container.size)}%`"
```

Comment:
container can just determine this property itself from the passed in properties.